scripts/versionsync.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+CHECK=0
+while getopts ":c" opt; do
+	case $opt in
+		c)
+			CHECK=1
+			;;
+		\?)
+			echo "Invalid option: -$OPTARG. Use -c to show changes without applying, use no options to apply changes." >&2
+			exit 1
+			;;
+	esac
+done
+
+# Version appearing earliest in CHANGELOGFILE will be used as ground truth.
```

Comment:
This makes sense to me since we have the changelog check in CI. That just checks if the changelog was updated, may be good to add a "did you update the version" check at some point too.